app/modules/tmr.c

```diff
@@ -184,16 +184,21 @@ static int tmr_register(lua_State* L){
 // Lua: tmr.start( id / ref )
```

Comment:
`Lua: tmr.start(ref, [restart])`?  More generally, if you're wanting to tidy `tmr`, could you drop the `id` from all of the `Lua:` comments?

Reply:
Ok, let's do it.